Source/Core/AudioCommon/CMakeLists.txt

```diff
@@ -33,8 +33,17 @@ endif(PULSEAUDIO_FOUND)
 
 if(WIN32)
 	set(SRCS ${SRCS} XAudio2Stream.cpp)
+
+	add_dolphin_library(audiocommon_xaudio27 "XAudio2_7Stream.cpp" "${LIBS}")
```

Comment:
None, really since you can't suppress requirements from a dependency.
This is a very special target unfortunately. Even in the Visual Studio solution it has special build settings too.